### ISSUE TITLE
wezterm ssh does not support Chinese input method

### ISSUE THREAD

**Issue by vvtommy** [wez/wezterm#2434]
### What Operating System(s) are you seeing this problem on?

macOS

### WezTerm version

20220807-113146-c2fee766

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

Cannot enter one Chinese word (composed of multiple Chinese characters) at a time

### To Reproduce

1. `wezterm --config-file "" ssh -v remote.host`
2. type a few chinese words(more than one charactor)
![image](https://user-images.githubusercontent.com/743350/185651401-73b214bc-b691-4419-bc81-f908a7ff9e29.png) 
3. nothing happaned🤷‍♂️

one new line of log appears：
```log
23:12:04.703  ERROR  mux::ssh                                   > boo
```

### Configuration

no config

### Logs

```
23:12:04.703  ERROR  mux::ssh                                   > boo
```

**Comment by wez** [comment]
I can't reproduce this with the latest nightly; looks like it got fixed by e6421d1b725715abe0463ea9040909de78c56563

**Comment by wez** [comment]
This should be fixed now in `main`.

It typically takes about an hour before fixes are available as nightly builds for all platforms.  Linux builds are the fastest to build and are often available within about 20 minutes.  Windows and macOS builds take a bit longer.

Please take a few moments to try out the fix and let me know how that works out. You can find the nightly downloads for your system in the [wezterm installation docs](https://wezfurlong.org/wezterm/installation.html).

If you prefer to use packages provided by your distribution or package manager of choice and don't want to replace that with a nightly download, keep in mind that you can download portable packages (eg: a `.dmg` file on macOS, a `.zip` file on Windows and an `.AppImage` file on Linux) that can be run without permanently installing or replacing an existing package, and can then simply be deleted once you no longer need them. 

If you are eager and can [build from source](https://wezfurlong.org/wezterm/install/source.html) then you may be able to try this out more quickly.

**Comment by vvtommy** [comment]
> I can't reproduce this with the latest nightly; looks like it got fixed by [e6421d1](https://github.com/wez/wezterm/commit/e6421d1b725715abe0463ea9040909de78c56563)

Sorry for wasting your time. I rechecked the latest nightly build(`20220818-135000-eb65b5fb`, eb65b5fb ), and confirmed that the problem **had been fixed**.

Maybe the wrong version was run yesterday when doing the nightly-build check 😺.

**Comment by wez** [comment]
It wasn't a waste of time: you reported a bug that I didn't know we had, even though I had fixed it without realizing it :)
